### PR TITLE
Expose IPV6_FLOWINFO* on Linux

### PR DESCRIPTION
**Pull request by Ralith**

### CODE REVIEW

File: libc-test/build.rs [hunk 1]
```diff
@@ -695,7 +695,18 @@ fn do_ctest() {
...
-            "AF_XDP" | "PF_XDP" | "SOL_XDP" if linux => true,
+            "AF_XDP"
+            | "PF_XDP"
+            | "SOL_XDP"
+            | "IPV6_FLOWINFO"
+            | "IPV6_FLOWLABEL_MGR"
+            | "IPV6_FLOWINFO_SEND"
+            | "IPV6_FLOWINFO_FLOWLABEL"
+            | "IPV6_FLOWINFO_PRIORITY"
+                if linux =>
```
Comment (Ralith) [inline libc-test/build.rs]: Unsure if these should be ignored for android too.

**Comment by gnzlbg** [comment]
Thank you!

**Pull request merged**